### DISCUSSION TITLE
OpenDTU kommerziell ausgeschlachtet?

### DISCUSSION THREAD

**Discussion by CrashCommando** [tbnobody/OpenDTU#784] [General]
Ich war gerade etwas am Preise vergleichen für einen Hoymiles Wechselrichter. Dabei ist mir auf diversen Plattformen aufgefallen, wie viele gewerbliche Anbieter mittlerweile fertige OpenDTUs auf dem Markt werfen (und die auch fleißig gekauft werden!), für die Software aber natürlich auf die Github Repos referenzieren. 
Wer sich die OpenDTU selbst zusammengebaut hat, weiß wohl, dass sich die Kosten und den Aufwand für den ESP32, NRF24L01+ und vielleicht noch ein (gedrucktes) Gehäuse eher in Grenzen halten. Die eigentliche Arbeit steckt in der Software. Einige User, allen voran @tbnobody stecken enorm viel Freizeit in das Projekt, aber andere schlagen daraus Kapital.
Ich weiß nicht, wie das die Community sieht, aber für mich hätte das irgendwie einen unangenehmen Beigeschmack. 

PS: Falls solche Grundsatz Diskussionen nicht erwünscht sind, bitte wieder löschen.

**Comment by DPO99** [comment]
Ich persönlich finde es bis zu dem Punkt okay, bis damit angefangen wird unverhältnismäßig viel Geld verdienen zu wollen. 

Materialpreis plus bisschen Arbeitszeit ist okay. Nicht jeder kann das oder traut sich das zu, sowas zusammenzubauen. Aber manche Preise sind tatsächlich sehr unverschämt.

Unterm Strich ist es eben OpenSource. Wenn man das nicht Lizenzieren möchte kann man da  nicht wirklich was dran machen.

**Comment by madmartin** [comment]
Ich sehe das auch so wie @DPO99 - es handelt sich hier um Open-Source-Software etc. und das muß man aushalten.

Jeder Interessent muß sich selbst überlegen ob das Angebotene das richtige Preis/Leistungsverhältnis hat, und wenn für MICH 50€ für eine fertige OpenDTU unverschämt viel Geld erscheinen, ist es für andere die einzige Möglichkeit ohne Cloud-Zwang seine Wechselrichter zu betreiben.

Natürlich muß man darauf achten daß die Regeln eingehalten werden, wie z.B. den Käufer auf die GPL verweisen und die Möglichkeit zum Download der Quellen mitgeben.

Wer hier neidisch wird, der kann ja selbst versuchen ein Business mit Hard+Software aus Open-Source Quellen aufzuziehen.
